skcosmo/selection/CUR.py

```diff
@@ -0,0 +1,218 @@
+from abc import abstractmethod
```

Comment:
This module needs a module docstring

Reply:
Still true!

Reply:
Done!

Reply:
I meant a module-level docstring for "CUR.py", not a comment for the import 😃

Reply:
Like a description of what is in this file, and why

Reply:
... that's very confusing. on it.